src/Microsoft.VisualStudio.Debugger.Interop.DAP/Microsoft.VisualStudio.Debugger.Interop.DAP.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.VisualStudio.Debugger.Interop.DAP
+{
+    /// <summary>
+    /// An extended set of evaluate flags that are in Debug Adapter Protocol but not listed in AD7
+    /// </summary>
+    [Flags]
+    public enum DAPEvalFlags
+    {
+        /// <summary>
+        /// No additional Eval Flags from DAP
+        /// </summary>
+        NONE = 0,
+        /// <summary>
+        /// Evaluation is for a clipboard context
+        /// </summary>
+        CLIPBOARD_CONTEXT = 1
+    }
+
+    /// <summary>
+    /// IDebugExpresssion for Debug Adapter Protocol
+    /// </summary>
+    [ComImport()]
+    [ComVisible(true)]
+    [Guid("76C710C4-CE66-4422-AC7C-1E41B3FC0BE3")]
+    [InterfaceType(1)]
+    public interface IDebugExpressionDAP
+    {
+        /// <summary>
+        /// Evaluate for IDebugExpressionDAP
+        /// </summary>
+        /// <param name="dwFlags"></param>
+        /// <param name="dapFlags"></param>
```

Comment:
Theres no point in having these `param` comments here if they are empty. Either add some text or remove them. Also explain why we want this in the summary.